selenium_tests/course_info/course_info_base_test_case.py

```diff
@@ -4,32 +4,54 @@
 
 from urlparse import urljoin
 from django.conf import settings
+from selenium_tests.course_info.page_objects.course_info_search_page_object import \
+    CourseSearchPageObject
 
 
 class CourseInfoBaseTestCase(BaseSeleniumTestCase):
-    """
-    Bulk Create base test case, all other tests will subclass this class
-    """
+
+    _search_page = None
```

Comment:
This seems to be used as an instance member, not a class member, I wouldn't define it up here.